### ISSUE TITLE
sigdev error when training model with image data generator

### ISSUE THREAD

**Issue by thebeancounter** [tensorflow/tensorflow#27002]
Hi 

see also [SO question](https://stackoverflow.com/questions/55283194/tensorflow-gpu-sigdev-error-when-training-on-image-data-generator) regarding this issue 

I am using ubuntu with conda, python 3.6 and tf gpu 1.12, and using image data generator to load images, and then fit generator. 

I am using this code 

    i = gen.flow_from_dataframe(dataframe=df, x_col="new_filename",
                                y_col="label", class_mode="sparse",
                                directory=links_dir, target_size=(224, 224), batch_size=32)
    
    model.fit_generator(i, epochs=1, workers=1,
                        use_multiprocessing=False, max_queue_size=1,
                        verbose=1)

and getting this error, without any batches even running 

> Process finished with exit code 139 (interrupted by signal 11:
> SIGSEGV)

I validated that the generator is loading the images using 

I used nvidia smi and made sure the model is created on GPU and the gpu memory is used.

The model is a simple, small conv net, nothing this GPU can't handle. 

print(gen[0]) and got data and everything went fine.
So I assume that the image data is fine.

What can be the issue here? could this be happening because I have two GPU's in the machine and I am only using one?

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow-tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

**Comment by thebeancounter** [comment]
@ymodak 
I suspect this is a bug, given it does not work as expected (I used the same code many times without any issues in the past)

**Comment by ymodak** [comment]
@thebeancounter Before I reopen the issue, can you please tell me what changes you made to your configuration that might have caused your code to crash since you were able to execute it successfully in the past ? Thanks!
